docker-hub/download-rate-limit.md

```diff
@@ -77,12 +78,25 @@ Which should return headers including these:
 ```http
 ratelimit-limit: 100;w=21600
 ratelimit-remaining: 76;w=21600
+docker-rateLimit-source: 0.0.0.0
 ```
 
-This means my limit is 100 per 21600 seconds (6 hours), and I have 76 pulls remaining.
+This means my limit is 100 per 21600 seconds (6 hours), I have 76 pulls remaining and I am limited on the ip 0.0.0.0 (not a valid ip address, used as an example).
 
 > Remember that these headers are best-effort and there will be small variations.
 
+### docker-RateLimit-Source
+
+This header will show what it used to limit you. It can help diagnose any issue.
+If you are not authenticated, it will show your ip address.
+If you are authenticated, it will show your `accountID`.
+
+You can compare this `accountID` with the one associated with your account by requesting the docker Hub API:
+```http
+https://hub.docker.com/v2/users/<namespace>/
+```
+(replace `<namespace>` with the account name)
```

Comment:
```suggestion
This header will show what source was used to limit you. It can help answer questions like "Am I authenticating correctly?" and "Is it rate limiting against my IP or my user account?"
If you are not authenticated, it will show your IP address.
If you are authenticated, it will show your `accountID`.

You can compare this `accountID` with the one associated with your account by requesting the docker Hub API:
```http
https://hub.docker.com/v2/users/<namespace>/
```
(replace `<namespace>` with the account name)
```